OptimizelySDK/Optimizely.cs

```diff
@@ -429,10 +429,16 @@ public bool IsFeatureEnabled(string featureKey, string userId, UserAttributes us
 
             if (decision != null)
             {
-                var experiment = Config.GetExperimentFromId(decision.ExperimentId);
+                // If experiment does not found in either experiment or rollout id map, return default value.
+                var experiment = Config.GetExperimentByIdFromAllExperiments(decision.ExperimentId);
```

Comment:
I don't prefer AllExperiments method. You should check decision, if its from experiment then get experiment, decision from rollout, get experiment from rollout.

Reply:
@oakbani

Reply:
Agreed

Reply:
Done